query/aql_compiler.go

```diff
@@ -1037,11 +1038,18 @@ func (qc *AQLQueryContext) Rewrite(expression expr.Expr) expr.Expr {
 					nil, "expect 1 argument to be a column for %s", e.Name)
 				break
 			}
+
 			e.Name = hllCallName
-			e.Args[0] = &expr.UnaryExpr{
-				Op:       expr.GET_HLL_VALUE,
-				Expr:     colRef,
-				ExprType: expr.Unsigned,
+			if colRef.IsHLLColumn {
+				// 1. column is hll column itself
+				e.Args[0] = colRef
```

Comment:
seems this branch is a noop. Is e.Args[0] already colRef?

Reply:
Done